src/popover/popover.spec.ts

```diff
@@ -382,6 +403,7 @@ describe('ngb-popover', () => {
       config = c;
       config.placement = 'bottom';
       config.triggers = 'hover';
+      config.container = 'body';
```

Comment:
Doesn't this apply to popover and not popoverwindow? Shouldn't it test that it takes custom config properly?